components/operations-controller/internal/config/config.go

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2020 The Compass Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"reflect"
+
+	"github.com/kyma-incubator/compass/components/operations-controller/internal/director"
+	"github.com/kyma-incubator/compass/components/operations-controller/internal/server"
+	"github.com/kyma-incubator/compass/components/operations-controller/internal/webhook"
+
+	"github.com/kyma-incubator/compass/components/system-broker/pkg/env"
+	"github.com/kyma-incubator/compass/components/system-broker/pkg/graphql"
+	"github.com/kyma-incubator/compass/components/system-broker/pkg/http"
+	"github.com/pkg/errors"
+	"github.com/spf13/pflag"
+)
+
+// Validatable defines entities capable of being validated
+type Validatable interface {
```

Comment:
`Validatable` is a pretty common interface and so you might see that there are other dependencies/libs which we use, which provide the same interface. While we could technically import their own version of this interface I think it's cleaner to redefine the interface on our own instead of importing and coupling our package with some other arbitrary one which happened to declare the same interface (ex.`import inputvalidation.Validatable`).
Also generally in Go I think it's accepted to redefine small constructs/types/interfaces instead of always living with the idea of reusing everything as much as possible. It makes the codebase simpler and easier to understand.